src/docs/patterns/forms.mdx

```diff
@@ -34,3 +37,45 @@ If your form fields have complex requirements for valid values, you can list the
 If there is an error with the data entered into a field, the field should indicate to the user that there is a problem. Label and helper text should be changed to a red color and a warning icon should appear in the text input. The error message should be displayed in the helper text location below the field (any existing helper text should be replaced with the error message).
 
 Error messages should be succinct and should not provide information that could compromise security (e.g., "Invalid Credentials" instead of "Incorrect Password").
+
+In general, there are two different types of input field verification: on blur, on change, on page change.
+
+### Verify Input on Blur
+
+In order to not "push" the user too much, most text fields should remain "verify on blur". That is, most of the time, you should only verify user input when they stop typing.
+
+<ImageGrid
+    images={[OnBlurImage]}
+    gridImageProps={{ sm: 12 }}
+    caption={[
+        '1: Do not check for validity or display error messages when the user is typing the desired text.',
+        '2: The user is finished and defocuses / “blurs away” from the input field. Now their input should be checked.',
+    ]}
+/>
```

Comment:
```suggestion
### Verifying on Blur

In order to avoid distracting or confusing a user, most text fields should verify input when the field loses focus.

<ImageGrid
    images={[OnBlurImage]}
    gridImageProps={{ sm: 12 }}
    caption={[
        '1: Do not check for validity or display error messages when the user is typing in the field.',
        '2: The user is finished and the input has lost focus (i.e., "blurred"). Now the input should be verified.',
    ]}
/>
```